graph/src/data/graphql/values.rs

```diff
@@ -0,0 +1,84 @@
+use failure::Error;
+use graphql_parser::query::Value;
+use std::collections::BTreeMap;
+use std::str::FromStr;
+
+use crate::web3::types::H160;
+
+pub trait TryFromValue: Sized {
```

Comment:
We can't replace this with `TryFrom`?

Reply:
I didn't do it because having a fixed `Error` type for all `TypedFromEntity` implementations seemed convenient. I'll try `TryFrom` again.

Reply:
As discussed in person, we're going to keep this for now, as it would be a lot of hassle to implement `TryFrom` in situations where both the source and target type are external, like with
```rust
impl TryFrom<&serde_json::Value> for String
```